### DISCUSSION TITLE
Offline report writing

### DISCUSSION THREAD

**Discussion by aronmolnar** [Syslifters/sysreptor#79] [Q&A]
It happens that pentesters need to write their reports when they have no Internet connection.

One solution that already works today is to install a community version at the pentester's device and to export the project as soon as it's finished and to import it at the main installation.

A major drawback is currently that there is no batch export of finding templates and project designs (this could be implemented).

Another solution would be to allow to transfer data and license directly from main installation to a local installation.

However, the implementation might be very complex (e. g. user management, do not share password hashes, map local users to original users, locking reports while written on a local installation, etc.), especially compared to the little benefits (SysReptor must still be installed locally, only import/export of projects/templates/designs could be simplified).

Maybe there are better/easier solutions? Any ideas?

**Comment by unixerius** [comment]
Honestly maybe the best alternative is to simply make notes in Markdown and later copy/paste the findings. Not ideal, but 🤷‍♀️.
